.github/workflows/main.yml

```diff
@@ -86,6 +86,7 @@ jobs:
           api-level: ${{ matrix.api-level }}
           target: ${{ matrix.target }}
           arch: ${{ matrix.arch }}
+          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none -no-snapshot-save
```

Comment:
Adding `-no-snapshot-save` here ensures that on a cache miss the newly created emulator is not polluted by test execution that comes right after it.